.circleci/config.yml

```diff
@@ -364,6 +388,11 @@ workflows:
       - upload_coverage:
           requires:
             - test_cover
+      - reproducible_builds:
+          filters:
+            branches:
+              only:
+                - master
```

Comment:
Are we not doing builds for release branches?

Reply:
We should. Master is being used for development now. `vX` are stable branches

Reply:
Will amend accordingly, thanks

Reply:
Done